### ISSUE TITLE
Automelt doesn't designate steel chests

### ISSUE THREAD

**Issue by kungyen0** [DFHack/dfhack#2982]
Using a stockpile for meltable steel furniture - all items seem to get designated for melting properly via automelt except for steel chests, which clutter up the stockpile over time.

**Comment by myk002** [comment]
There is specific code in automelt that skips chests that I haven't been able to determine the reason for. I believe we can just remove the check (or maybe replace it with a check that prevents *filled* chests from being melted?)

@eamondo2 thoughts?

**Comment by quietust** [comment]
My guess is that it was a workaround for [DF bug #2493](https://dwarffortress.mantishub.io/view.php?id=2493) - prior to version 0.40.05, designating metal chests for melting would cause them to perpetually clutter the smelter (and waste time and possibly also fuel) without actually melting them.

**Comment by eamondo2** [comment]
Yeah I'd lifted the flag checking code from the existing automelt implementation.

**Comment by myk002** [comment]
hilarious. yes, that makes sense, and good to have confirmation that removing the check is likely ok now
